.gitlab-ci.yml

```diff
@@ -30,6 +31,33 @@ linux-test:
   tags:
     - linux-docker
 
+lerna-patch:
+  stage: release
+  script:
+    - yarn lerna:bump patch
+    - yarn lerna:publish 
+  only:
+    - master
+    # - /^\w.*-release-patch.*$/
+
+lerna-minor:
+  stage: release
+  script:
+    - yarn lerna:bump minor
```

Comment:
Do we need a yarn install here? Or does this run after a previous job that already did yarn install?

Reply:
fwiu the stages run in order as they are defined, i.e. test, release, dockerize